packages/cli/src/collect/collect.js

```diff
@@ -39,6 +39,10 @@ function buildCommand(yargs) {
       description:
         'A URL to run Lighthouse on. Use this flag multiple times to evaluate multiple URLs.',
     },
+    filterUrl: {
```

Comment:
WDYT about... 

`ignoreAutodiscoverUrl`
`skipAutodiscoverUrl`
`autodiscoverUrlBlocklist`
`autodiscoverSkipUrl`
`autodiscoverIgnoreUrl`

`skipAutodiscoverUrl` and `autodiscoverUrlBlocklist` are probably my personal favorites.

rationale:

- `autodiscover` feels like it needs to be in there somewhere to signify it only works for autodiscovered URLs (comment isn't totally accurate atm, since manually specified URLs can still be used with `staticDistDir`)
- `filterUrl` makes me think it's a regex pattern for *allowing* certain URLs through, `ignore`/`skips`/`block` are more explicit